src/main/components/DataTree.jsx

```diff
@@ -4,7 +4,8 @@ import JsonTree from '../lib/treeView/jsonTree';
 import '../lib/treeView/jsonTree.css';
 
 const DataTree = (props) => {
-  const { treeCount, data } = props;
+  const { treeCount, data, options } = props;
```

Comment:
not gonna hold up anything but if when you're refactoring maybe rename `options`?  i'm thinking it's `dataTreeOptions` or something like that.  I didn't follow initially

Reply:
Yeah not sure what the best approach is here. I agree it’s not the clearest name but we’re naming a lot of things `DatatTree` which gets really long, too… Something to brainstorm later.